components/Modal.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, TouchableOpacity } from 'react-native';
+
+
+export default class App extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {isGoingToStore: false, isCheckingOut: false, isEnRoute: false, isDelivered: false};
```

Comment:
Format you dictionaries like [this](https://github.com/airbnb/javascript#commas--leading-trailing)